tests/e2e/workloads/test_create_scale_pods_and_pvcs_using_kube_job.py

```diff
@@ -88,10 +90,32 @@ def test_create_scale_pods_and_pvcs_using_kube_job_ms(
         config.switch_to_consumer()
         log.info("The resources created successfully using the kube job")
 
+    @skipif_ms_provider_and_consumer
```

Comment:
Why do we need to skip this when both the provider and consumer are present ? Instead, I think, this test case can use the consumer alone for running.

Reply:
The idea behind this is that we want to run some of the tests only with provider and consumer, but sometimes we want to cover the same tests above(with provider and consumer only) to run with a single cluster. 

For example, if we have one test with AWS and we want to create a similar test with vSphere - there is no need to run the second test with AWS, as we covered in the first test.

Reply:
Are we going to regularly (in CI) run MS related jobs in single cluster scenario?

Reply:
I saw that in tier1, we run the tests with one provider or consumer. I didn't see a run with multiple consumers and no provider - but we also should consider this scenario